setup.cfg

```diff
@@ -1,2 +0,0 @@
-[bdist_rpm]
-requires =  python-requests >= 2.5.0
```

Comment:
I see a new dependency for deb in stdeb.cfg, but this change appears to remove the dependency for rpm.  Is that expected and correct?